eclass/git-r3.eclass

```diff
@@ -241,6 +241,13 @@ fi
 # EGIT_SUBMODULES=( '*' '-test-*' test-lib )
 # @CODE
 
+# @ECLASS_VARIABLE: EGIT_NO_FETCH
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# Don't fetch data from remote repository
+#
+# EGIT_NO_FETCH=1|yes|true
```

Comment:
That's not the syntax we use.

Reply:
I can always change it to true if not empty version if approved.

Reply:
I first saw it used in `ruby-ng.eclass` which I thought makes sense and started using it with the ebuilds I modify/make myself.